src/nautiluszim/scraper.py

```diff
@@ -499,6 +503,10 @@ def process_collection_entries(self):
                     is_front=False,
                 )
 
+                if uri.startswith("http") and self.download_delay and index < num_files - 1:
```

Comment:
It would be more elegant to move this to the beginning of the loop and change the condition to `index > 0` (i.e. do not pause on first item). Especially since we already have a `if uri.startswith("http")` there.